CMakeLists.txt

```diff
@@ -651,8 +651,8 @@ add_subdirectory(tools)
 ################################################################################
 # Testing
 ################################################################################
-option(ENABLE_UNIT_TESTS "Enable unit tests" ON)
-option(ENABLE_SYSTEM_TESTS "Enable system tests" ON)
+option(ENABLE_UNIT_TESTS "Enable unit tests" OFF)
+option(ENABLE_SYSTEM_TESTS "Enable system tests" OFF)
```

Comment:
I don't think these should be off by default. It discourages users from testing. Instead we should make enabling testing simpler.